tests/jballerina-unit-test/src/test/resources/test-src/annotations/annot_attachments_negative.bal

```diff
@@ -975,3 +975,7 @@ type F4 record {|
 type F5 record {|
     int x;
 |};
+
```

Comment:
Shall we add a test at module-level also?

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/41794/commits/f02002e6d47d9020fc00dc71921218af8b2839f5